quilt/tools/command.py

```diff
@@ -200,13 +200,14 @@ def log(session, package):
             pkg=pkg
         )
     )
-
+    # table header
     format_str = "%-64s %-19s %s"
-
-    print(format_str % ("Hash", "Created", "Author"))
-    for entry in response.json()['logs']:
+    print(format_str % ("Hash", "Pushed", "Author"))
+    # table body
+    logs = sorted(response.json()['logs'], key=lambda x: parse_utc(x['created']), reverse=True)
```

Comment:
The server already sorts this by time (and by ID, in case of duplicate time), so it's kind of silly to sort it again. I'd rather change the server code - or just reverse the order here.